config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -0,0 +1,678 @@
+# generated by gen-jobs.sh, do not edit manually
+
+presubmits:
+  kubernetes-csi/csi-driver-host-path:
+  - name: pull-sig-storage-csi-driver-host-path-1-13-on-kubernetes-1-13
+    # Experimental job, explicitly needs to be started with /test.
+    # This can be enabled once the components have the necessary configuration
+    # for this combination of deployment+Kubernetes.
+    always_run: false
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - ./.prow.sh
+        env:
+        # We pick some version for which there are pre-built images for kind.
+        # Update only when the newer version is known to not cause issues,
+        # otherwise presubmit jobs may start to fail for reasons that are
+        # unrelated to the PR. Testing against the latest Kubernetes is covered
+        # by periodic jobs (see https://k8s-testgrid.appspot.com/sig-storage-csi#Summary).
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: "1.13.3"
+        - name: CSI_PROW_KUBERNETES_DEPLOYMENT
+          value: "1.13"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 2000m
+  - name: pull-sig-storage-csi-driver-host-path-deployment-1-13-on-kubernetes-1-13
```

Comment:
what is the difference between this and `pull-sig-storage-csi-driver-host-path-1-13-on-kubernetes-1-13` above?

Reply:
The difference is explained in https://github.com/kubernetes/test-infra/blob/1f2e5650f0475f69006c62705e2881938ef7825c/config/jobs/kubernetes-csi/gen-jobs.sh#L143-L148

One tests with updated images, the other the images specified in the deployment.

Reply:
Why do the jobs that don't modify the images need to run in the pull job?

Reply:
I'm still unclear why do we need to run jobs that don't modify images in pull jobs?

Reply:
That is for PRs that modify just the `deployment` directory. Suppose I submit a PR with `image: quay.io/k8scsi/hostpathplugin:v1.1.0` in `deploy/kubernetes-1.13` (something we talked about yesterday) and that image doesn't exist yet. If merged, the deployment in master gets broken. This is bad because we point people to the master branch as the source for the latest stable deployments. We have to do that a) because a tagged release of csi-driver-host-path cannot contain a .yaml file with the image for that tagged release (chicken-and-egg problem) b) it's easier to have one link that remains constant (link to master vs. link to latest release).

One could argue that it's okay to break master temporarily, but according to Murphy, this time window will be when someone tries out the deployment and notices the regression. I'd prefer to keep this as pre-submit. We can limit the number of times that it actually runs by adding a path filter. That should also clarify the purpose. How about that?

Reply:
I would prefer having the tagged version actually reference the correct deployment images.  If someone is automatically deploying the driver in some production environment, they should be depending on a tagged version, not master.

Reply:
> I would prefer having the tagged version actually reference the correct deployment images.

Then how do you propose to deal with the chicken-and-egg problem? Allow breaking master temporarily?

>  If someone is automatically deploying the driver in some production environment, they should be depending on a tagged version, not master.

This is the hostpath driver. It's never going to be deployed in production.

Reply:
Yes, breaking master temporarily

Reply:
In that case we can remove these jobs again.

Reply:
I've removed them the PR.